javascript/react/src/components/HistoryPanel.types.ts

```diff
@@ -14,5 +14,6 @@ export interface HistoryPanelProps {
 	workflow: Workflow
 	title?: string
 	order?: DisplayOrder
+	stepsCount?: number
```

Comment:
This seems like a slightly weird expectation since the data is in the workflow. Maybe just a boolean like `showStepCount` and then still retrieve it from the workflow internally?